functions/Test-DbaLastBackup.ps1

```diff
@@ -175,7 +184,9 @@ function Test-DbaLastBackup {
         [string]$AzureCredential,
         [parameter(ValueFromPipeline)]
         [Microsoft.SqlServer.Management.Smo.Database[]]$InputObject,
-        [switch]$EnableException
+        [switch]$EnableException,
```

Comment:
Please move the additional parameters before EnableException. This one should always be the last parameter. In both help and the param block.

Reply:
i have moved them up as requested, and added the parameters to the test script.